### ISSUE TITLE
Rolling a weapon from the item tab in a npc sheet doesn't get the number of dice right.

### ISSUE THREAD

**Issue by parvusdomus** [Foundry-VTT-Lex-Arcana/foundryvtt-lexarcana#110]
We need to either recognize the number of dice the peritia is assigned to or/and remove that roll icon in the meantime.

**Comment by parvusdomus** [comment]
Not sure if this can be considered a bug or a FR, it can be labeled NPC sheet improvement?

**Comment by pplans** [comment]
This might need some thinking to avoid creating non needed data or workarounds that will make the system too complex to maintain

**Comment by parvusdomus** [comment]
Yes, absolutelly. I'm thinking on removig the skill roll from the item, and only leave the damage roll for now.

**Comment by pplans** [comment]
please don't, I had some ideas with that. I need still sometime as I am busy with my health issues
